about.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="assets/css/styles.css">
+  <!-- Cocogoose font family -->
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <!-- Lato font family -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
+  <title>Art Selection welcome page</title>
+</head>
+<body>
+  <header>
+    <!--ADD THE FIRST LEVEL -->
+    <div class="container-fluid display-desktop social" id="social">
+      
+    </div>
+    <!--ADD THE NAVBAR -->
+    <nav class="navbar navbar-expand-md navbar-light bg-light sticky-top" id="my-navbar">
+      
+    </nav>
+  </header>
+  <main>
+
+
+    <!-- ABOUT SECTION FOR PAGE 2-->
+    <div class="container mt-5 py-4">
+      <div class="container-fluid">
+        <div class="row about text-center">
+          <div class="col-12">
+            <h1 class="display-4 mb-3">ANNUAL SMSE SUMMIT 2023</h1>
+            <hr class="full-width mx-auto mb-5">
+            <div class="col-12">
+              <p class="alert-info lead p-4"> <!-- trying different classes //class="lead"-->
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+                Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi 
+                ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit 
+                in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+                Excepteur sint occaecat cupidatat non proident, 
+                sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
- [ ] It is required to stick to the guiding template as much as possbile.  Text color and the background color on your design does not tally with the text color and the background color on the original design.
- [ ] You might also need to consider the width of the box.

### Your Design
![your-about-first-part](https://user-images.githubusercontent.com/26597432/235004007-cae37b29-f619-4e63-bcb1-5656f14129b9.png)

### Guiding Template
![temp-about-first-part](https://user-images.githubusercontent.com/26597432/235004025-abd97b1a-122b-4acb-9f8a-6381a70c1045.png)

Reply:
@Qoosim Box size fixed